docs/integrate/rpc-node-providers/omnia.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Omnia Protocol"
+sidebar_position: 1
+---
+
+## Introduction
+
+[OMNIA Protocol](https://omniatech.io) is a blockchain infrastructure provider that delivers optimal performance without sacrificing privacy, security or compliance.
+
+## Privacy and Consumer Protection
+
+Transactions can be tracked even before they get mined, compromising privacy in the face of malicious actors who track your behavior. OMNIA prevents this by obfuscating your off-chain data, thus safeguarding you against scams, hacks, and front-running attacks. All the data and metadata remain private to the users. No third party is able to access, analyze or track it. OMNIA leverages different technologies and approaches to guarantee the privacy of their users, from front-running protection and private mempools, to obfuscation and random dispatching.
```

Comment:
We're proof-of-stake blockchain - therefore we need to change "mined" here :)

Reply:
Got it, I will fix this.